cli/src/list.rs

```diff
@@ -198,27 +253,40 @@ pub async fn list(current_crate: &Crate, is_verbose: bool) -> eyre::Result<()> {
         parser.visit_file(&syntax);
     }
 
+    if !is_verbose {
+        simple(&parser.functions, current_crate);
+        return Ok(());
+    }
+
+    let domain = format!("https://{}.usekinetics.com", current_crate.name);
+
     let mut endpoint_rows = Vec::new();
     let mut cron_rows = Vec::new();
     let mut worker_rows = Vec::new();
+    let client = Client::new(false)?;
 
     for parsed_function in parser.functions.clone() {
         let func_name = parsed_function.func_name(false);
         let func_path = parsed_function.relative_path.clone();
```

Comment:
@quesabe can we convert the `status()` function to method of [`Function`](https://github.com/ottofeller/kinetics/blob/cb0b9dccb14c2d2d97f97c6e4bffff164c723303/cli/src/function.rs#L32)?